guides/v2.2/frontend-dev-guide/css-topics/css_debug.md

```diff
@@ -12,6 +12,7 @@ The topic describes how to install, configure, and use [Grunt JavaScript task ru
 - Make sure you [set]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html) your Magento application to the developer or default [mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html).
 - Install and configure Grunt as described in [Using Grunt for Magento tasks]({{ page.baseurl }}/frontend-dev-guide/tools/using_grunt.html)
 
+The default mode sets the Less compilation mode to Server-side Less compilation.
```

Comment:
Can you add this as a separate sentence right after the sentence that begins on line 12 rather than its own line?

Reply:
Hello @erikmarr 
Updated as per the suggestions.